Wrangler error related to `logpush = true`

I'm getting the following error running `wrangler deploy` when logpush is set to `true`. Logpush is not available to non-enterprise plan, maybe it could be documented. The error message doesn't tell the actual issue.

```
✘ [ERROR] A request to the Cloudflare API (/accounts/***/workers/scripts/r2-registry) failed.

  workers.api.error.unauthorized [code: 10023]
```

Related to wrangler: cloudflare/workers-sdk#2991

--- a/wrangler.toml
+++ b/wrangler.toml
@@ -4,7 +4,6 @@ workers_dev = true
 main = "./index.ts"
 compatibility_date = "2022-04-18"
 compatibility_flags = ["streams_enable_constructors"]
-logpush = true
 
 ## Production
 [env.production]

